docs/source/Container.rst

```diff
@@ -0,0 +1,271 @@
+
+The out-of-the-box Land DA case described in this User's Guide builds a weather forecast for January 1, 2016 at 18z to January 3, 2016 at 18z. 
+
+.. attention::
+
+   This chapter of the User's Guide should **only** be used for container builds. For non-container builds, see :numref:`Chapter %s <BuildRunLandDA>`, which describes the steps for building and running Land DA on a :ref:`Level 1 System <LevelsOfSupport>` **without** a container. 
+
+.. _Prereqs:
+
+Prerequisites 
+*****************
+
+The containerized version of Land DA requires: 
+
+   * `Installation of Singularity <https://docs.sylabs.io/guides/latest/user-guide/quick_start.html>`__
+   * At least 6 CPU cores
+   * An **Intel** compiler and :term:`MPI` (available for free `here <https://www.intel.com/content/www/us/en/developer/tools/oneapi/hpc-toolkit-download.html>`__) 
+
+
+Install Singularity
+======================
+
+To build and run Land DA using a Singularity container, first install the Singularity package according to the `Singularity Installation Guide <https://docs.sylabs.io/guides/latest/user-guide/quick_start.html#quick-installation-steps>`__. This will include the installation of dependencies and the installation of the Go programming language. SingularityCE Version 3.7 or above is recommended. 
+
+.. note:: 
+   Docker containers can only be run with root privileges, and users generally do not have root privileges on :term:`HPCs <HPC>`. However, a Singularity image may be built directly from a Docker image for use on the system.
+
+.. _DownloadContainer:
+
+Build the Container
+**********************
+
+.. _CloudHPC:
+
+Set Environment Variables
+=============================
+
+For users working on systems with limited disk space in their ``/home`` directory, it is important to set the ``SINGULARITY_CACHEDIR`` and ``SINGULARITY_TMPDIR`` environment variables to point to a location with adequate disk space. For example:
+
+.. code-block:: 
+
+   export SINGULARITY_CACHEDIR=/absolute/path/to/writable/directory/cache
+   export SINGULARITY_TMPDIR=/absolute/path/to/writable/directory/tmp
+
+where ``/absolute/path/to/writable/directory/`` refers to a writable directory (usually a project or user directory within ``/lustre``, ``/work``, ``/scratch``, or ``/glade`` on NOAA :term:`RDHPCS` systems). If the ``cache`` and ``tmp`` directories do not exist already, they must be created with a ``mkdir`` command. 
+
+On NOAA Cloud systems, the ``sudo su`` command may also be required:
```

Comment:
This is only required if a user is building the container with sudo (e.g. sudo singularity build ...)

Reply:
OTOH. Nobody but us is building or running on the noaacloud, so this isn't a huge issue....

Reply:
And the other question is whether the build will work on NOAA Cloud without `sudo`... we've had issues in the past. Do you want me to add `sudo` to the build command and just note that not all users will need it? Currently, I do the opposite (leave of `sudo` but note that some users might need it).

Reply:
The build should work there without sudo, but it is true that there are platforms where sudo may still be required. I think the key point is that if you are going to build with sudo, you should sudo su and then set the environment variables as root OR I think "sudo -E singularity build ...." will work and preserve the user's environment variables if the TMPDIR and CACHEDIR were set under the user's name.

Reply:
Ok, I think I've addressed this, but let me know if I missed anything.